inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/ConceptFeatureEditor.java

```diff
@@ -289,6 +291,17 @@ protected void onInitialize() {
     public Component getFocusComponent() {
         return focusComponent;
     }
+    
+    private List<KBHandle> distinctByIri(List<KBHandle> aHandles)
+    {
+        Map<String, KBHandle> hMap = new HashMap<>();
+        for (KBHandle h: aHandles) {
+            if (!hMap.containsKey(h.getIdentifier())) {
```

Comment:
Is this *if* necessary? Iris would still be distinct if we just use *hMap.put(h.getIdentifier(), h)*, since HashMap doesn't allow duplicate keys.

Reply:
yeah, its not. actually, I was playing around with some else if but removed it later on.